### PR TITLE
Top bar changes

### PR DESCRIPTION
**Pull request by PiperLawson**
For cprod only: Made light background white. Added border to white background

For cprod and teams: Removed shadow. Added height.

### CODE REVIEW

File: lib/atomic/spacing.less [hunk 1]
```diff
@@ -26,6 +26,7 @@ body {
     --su-static24: 24px;
     --su-static32: 32px;
     --su-static48: 48px;
+    --su-static56: 56px;
```
Comment (dancormier) [inline lib/atomic/spacing.less]: We can calculate this value with existing variables by doing something like `calc(var(--su-static48 ) + var(--su-static8))` to keep the scale consistent
```suggestion
```

File: lib/exports/constants-colors.less [hunk 2]
```diff
@@ -242,8 +242,8 @@
     --theme-tag-hover-border-color: transparent;
 
     // Topbar
-    --theme-topbar-height: var(--su-static48);
-    --theme-topbar-background-color: var(--black-025);
+    --theme-topbar-height: var(--su-static56);
```
Comment (dancormier) [inline lib/exports/constants-colors.less]:
```suggestion
    --theme-topbar-height: calc(var(--su-static48 ) + var(--su-static8)); // 56px
```

**Review by dancormier — CHANGES_REQUESTED** [review]
> For cprod only: Made light background white. Added border to white background
> 
> For cprod and teams: Removed shadow. Added height.


All of these changes will apply everywhere. Is that your intention @PiperLawson?

<details>
<summary>Before vs After</summary>

### Before
image

### After
image

</details>

I believe this will be a breaking change in Core since page top padding is relative to topbar height. It's not a _huge_ breaking change but we'd need to make sure it is addressed when this ships.

[Giamir beat me to mentioning it](https://github.com/StackExchange/Stacks/pull/1360#pullrequestreview-1424742107): We format commit messages (which are prefilled by the PR title) using [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting. This PR might be merged with a commit message like `fix(topbar): update background and spacing`. I use [this cheat sheet](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index) to jog my terrible memory.

**Comment by giamir** [comment]
>  It's not a huge breaking change but we'd need to make sure it is addressed when this ships.

I think it might require some thorough testing in Core because as far as I remember I saw banners being placed into the page based on the fixed height of the topbar. Certainly mergeable but would require some extra eyes. 👀

**Comment by PiperLawson** [comment]
Can you explain this to me: 
> top padding is relative to topbar height

Where can I see how this might change things?

**Comment by dancormier** [comment]
> Can you explain this to me:
> 
> > top padding is relative to topbar height
> 
> Where can I see how this might change things?

I had this a little backwards.

This is kinda similar to the background color variable where it's exposed via the theming API. Themes could override this value by setting the `--theme-topbar-height` variable, _but_ we have [this shim file](https://github.com/StackEng/StackOverflow/blob/main/StackOverflow/Content/less/views/_topbar-stacks-shim.less) that does the overriding instead. FWIW, _I think_ is there to account for announcement banners and similar items that can affect page layout.

**Pull request merged**